cyences_app_for_splunk/bin/cs_product_list.py

```diff
@@ -131,7 +131,7 @@ def build_source_reviewer_search(by, values, first_call=True):
                 "macro_name": "cs_sophos",
                 "label": "Sophos Endpoint Protection Data",
                 "search_by": "sourcetype",
-                "search_values": "sophos_events",
```

Comment:
Create a separate PR for documentation changes for this data collection and upgrade guide.